site-src/implementations.md

```diff
@@ -318,6 +328,8 @@ The [LiteSpeed Ingress Controller](https://litespeedtech.com/products/litespeed-
 
 ### NGINX Kubernetes Gateway
 
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v0.7.1-NGINX Kubernetes Gateway-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v0.7.1/nginxinc-nginx-kubernetes-gateway.yaml)
```

Comment:
@shaneutt this badge isn't rendering

Reply:
Weird... I'm seeing it in the preview?

![image](https://github.com/kubernetes-sigs/gateway-api/assets/5332524/474d1d86-52d9-4ba9-b690-e03f4be15127)

Reply:
Ah ok. It's not showing in the rich diff like the other badges, but if it's in the preview that's good 👍